charts/operator/values.yaml

```diff
@@ -0,0 +1,26 @@
+image:
+  repository: "us.gcr.io/rookout/rookout-k8s-operator"
+  tag: "1.0"
+  pullPolicy: Always
+
+operator:
+  matchers:
+    - deployment : "example_deployment_matcher"
+      container: "example_container_matcher"
+      labelels:
+        - name: "deployment label name"
+          value: "deployment label value"
+      env_vars:
+        - name: "TEST_VAR"
+          value: "TEST_VALUE"
+
+        - name: "ROOKOUT_TOKEN"
+          value: "<YOUR TOKEN>"
```

Comment:
remove it, so if someone will run it wont run with this fake env var.
look at the DOP helm chart to see how we enforce a variable